shopfloor/services/zone_picking.py

```diff
@@ -205,11 +205,13 @@ def _data_for_select_picking_type(self, zone_location, picking_types):
     def _picking_type_zone_lines(self, zone_location, picking_type):
         return self.env["stock.move.line"].search(
             [
-                ("location_id", "=", zone_location.id),
                 # we have auto_join on picking_id
                 ("picking_id.picking_type_id", "=", picking_type.id),
                 ("qty_done", "=", 0),
                 ("state", "in", ("assigned", "partially_available")),
+                "|",
+                ("location_id", "=", zone_location.id),
+                ("location_id", "child_of", zone_location.id),
```

Comment:
`child_of` includes `=` no? We could remove the `=` criteria

Reply:
```python
>>> a = env["stock.location"].create({"name": "A", "barcode": "A"})
>>> b = env["stock.location"].create({"name": "B", "barcode": "B", "location_id": a.id})
>>> env["stock.location"].search([("id", "child_of", a.id)]).mapped("name")
['A', 'B']
```

Reply:
ohhh you are right, forgot it...

Reply:
fixed